specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2021-04-01-preview/apimanagement.json

```diff
@@ -628,12 +628,12 @@
       "description": "The name of the resource group.",
       "x-ms-parameter-location": "method"
     },
-    "SchemaIdParameter": {
+    "ApiSchemaIdParameter": {
```

Comment:
Does it make sense to reuse SchemaIdParameter for both service level and api level schemas?